pkg/search/result.go

```diff
@@ -82,10 +110,47 @@ type License struct {
 
 type User struct {
 	GravatarID string `json:"gravatar_id"`
-	ID         int64  `json:"id"`
+	ID         string `json:"node_id"`
 	Login      string `json:"login"`
 	SiteAdmin  bool   `json:"site_admin"`
 	Type       string `json:"type"`
+	URL        string `json:"url"`
+}
+
+type Issue struct {
+	Assignees         []User           `json:"assignees"`
+	Author            User             `json:"user"`
+	AuthorAssociation string           `json:"author_association"`
+	Body              string           `json:"body"`
+	ClosedAt          time.Time        `json:"closed_at"`
+	CommentsCount     int              `json:"comments"`
+	CreatedAt         time.Time        `json:"created_at"`
+	HTMLURL           string           `json:"html_url"`
+	ID                string           `json:"node_id"`
+	Labels            []Label          `json:"labels"`
+	IsLocked          bool             `json:"locked"`
+	Number            int              `json:"number"`
+	PullRequestLinks  PullRequestLinks `json:"pull_request"`
+	RepositoryURL     string           `json:"repository_url"`
+	State             string           `json:"state"`
+	Title             string           `json:"title"`
+	URL               string           `json:"url"`
```

Comment:
When requesting `--json url`, in other commands we print the `html_url` value for web browsers, not the API URL value, and so I think we should be the same here. This property could map to `html_url`, and you can remove the above HTMLURL field.